README.md

```diff
@@ -70,6 +70,22 @@ Compress the generated 'sn-table-ext' folder into the 'application/zip' file for
 | :-----------------------------------------------------------------------------: | :--------------------------------------------------------------: | :------------------------------------------------------------------------: |
 | Copy sn-table-ext into https://your-tenant.us.qlikcloud.com/console/extensions/ | Copy sn-table-ext into Qlik Management Console (QMC)->Extensions | Copy sn-table-ext into ..\Users\<UserName>\Documents\Qlik\Sense\Extensions |
 
+## React Native
+
+### Pre-reqs
```

Comment:
We probably want to add that you need to run in vscode

Reply:
I removed the .vscode file.